rest/src/main/kotlin/builder/channel/VoiceChannelCreateBuilder.kt

```diff
@@ -27,15 +28,15 @@ class VoiceChannelCreateBuilder(var name: String) : AuditRequestBuilder<GuildCha
     private var _position: OptionalInt = OptionalInt.Missing
     var position: Int? by ::_position.delegate()
 
-    val permissionOverwrites: MutableList<Overwrite> = mutableListOf()
+    override var permissionOverwrites: MutableSet<Overwrite>? = mutableSetOf()
```

Comment:
> Because it needs to be nullable in the modify builders

If something doesn't apply to a class, then it shouldn't have to implement it.

Replace the property with an `addOverwrite(Overwrite)` function and delegate to that instead. None of the DSLs require the removal or modification of overwrites anyhow.